action.yml

```diff
@@ -32,6 +32,14 @@ inputs:
   linting:
     description: "Toggle Helm linting, can be disabled by setting it to 'off'"
     required: false
+  commit_username:
+    description: "The user name used for the commit user"
+    required: false
+    default: "$GITHUB_ACTOR"
```

Comment:
Does GH Actions support env vars here?

Reply:
No it doesn't, just wanted to represent what the default value would be.

Reply:
Nevermind i think it does, let me test it out.

Reply:
It does, updated the PR as discussed